server/test/action_test.dart

```diff
@@ -44,15 +36,5 @@ void main() {
 
       expect(actionProto, equals(validActionProto));
     });
-
-    test('run() changes the line item status using the client', () async {
-      final status = 'ENTITY_STATUS_'
```

Comment:
Why do you delete this test?

Reply:
The run method got moved to `dv360.dart` but I forgot to delete it here.